core/primitives/src/stateless_validation/contract_distribution.rs

```diff
@@ -328,13 +349,16 @@ pub struct ChunkContractDeploys {
 }
 
 impl ChunkContractDeploys {
-    pub fn compress_contracts(contracts: &Vec<CodeBytes>) -> std::io::Result<Self> {
-        CompressedContractCode::encode(&contracts)
+    pub fn compress_contracts(contracts: Vec<ContractCode>) -> std::io::Result<Self> {
```

Comment:
I would prefer to leave `ChunkContractDeploys` api to work with the types it stores internally and leave conversion to and from `ContractCode` for the client code

Reply:
done